gapis/api/vulkan/transform_mapping_exporter.go

```diff
@@ -43,20 +43,24 @@ type mappingHandle struct {
 }
 
 type mappingExporter struct {
-	mappings       *map[uint64][]service.VulkanHandleMappingItem
-	thread         uint64
-	path           string
-	traceValues    []mappingHandle
-	notificationID uint64
+	mappings         *map[uint64][]service.VulkanHandleMappingItem
+	thread           uint64
+	path             string
+	traceValues      []mappingHandle
+	notificationID   uint64
+	usedFrameBuffers map[uint64]bool
```

Comment:
Sets in go are `map[<keytype>]struct[}`. Otherwise it's expected that the map may contain both `true` and `false` values.